core/templates/dev/head/pages/exploration_editor/simple_editor_tab/SimpleEditorBodyDirective.js

```diff
@@ -45,13 +62,19 @@ oppia.directive('simpleEditorBody', [
           $scope.canAddNewQuestion = (
             SimpleEditorManagerService.canAddNewQuestion);
           $scope.addState = SimpleEditorManagerService.addState;
+          $scope.changeQuestion = SimpleEditorManagerService.changeQuestion;
           $scope.addNewQuestion = SimpleEditorManagerService.addNewQuestion;
           $scope.canTryToFinishExploration =
             SimpleEditorManagerService.canTryToFinishExploration;
 
           $scope.getSubfieldId = function(question, label) {
             return QuestionIdService.getSubfieldId(question.getId(), label);
           };
+
+          $scope.setModalVisibility = function(value) {
```

Comment:
Instead of passing in `True`/`False` which breaks abstraction barriers, there should be two functions: `setModalHidden` and `setModalVisible` which will make the modal hidden or visible.